libs/langchain/langchain/document_loaders/pdf.py

```diff
@@ -335,21 +339,18 @@ def __init__(
             should_clean_pdf: a flag to clean the PDF file. Default is False.
             **kwargs: additional keyword arguments.
         """
-        super().__init__(file_path)
         self.mathpix_api_key = get_from_dict_or_env(
             kwargs, "mathpix_api_key", "MATHPIX_API_KEY"
         )
         self.mathpix_api_id = get_from_dict_or_env(
             kwargs, "mathpix_api_id", "MATHPIX_API_ID"
         )
+        headers = {"app_id": self.mathpix_api_id, "app_key": self.mathpix_api_key}
```

Comment:
@eyurtsev I think this is headers for api call not for file_path requests

Reply:
good catch! will update (just need to rename the existing property in that case)